src/App.jsx

```diff
@@ -1,37 +1,48 @@
 import React, { useEffect } from 'react';
-import { Route, Routes } from 'react-router-dom';
+import { Route, Routes, useNavigate } from 'react-router-dom';
 import { useDispatch, useSelector } from 'react-redux';
 import SignUp from './components/SignUp';
 import Home from './components/Home';
 import Index from './components/Index';
 import Reservations from './components/Reservations/Reservations';
 import Reserve from './components/Reserve';
 import Login from './components/Login';
+import AddHotel from './components/AddHotel';
 import { login } from './redux/login/login';
-import { getAllRoomsAsync } from './redux/room/room';
+import { getAllHotelsAsync } from './redux/hotel/hotel';
+import { getAllCitiesAsync } from './redux/city/city';
 import LogedUsers from './components/accessibility/LogedUsers';
 import DetailsView from './components/Details/DetailsView';
 
 const tokenKey = 'token';
 
 const App = () => {
   const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const loggedIn = useSelector((state) => state.login.isLoggedIn);
 
   useEffect(() => {
     if (localStorage.getItem(tokenKey)) {
       dispatch(login(JSON.parse(localStorage.getItem(tokenKey))));
     }
 
-    dispatch(getAllRoomsAsync());
+    dispatch(getAllHotelsAsync());
+    dispatch(getAllCitiesAsync());
   }, []);
 
-  const loggedIn = useSelector((state) => state.login.isLoggedIn);
+  useEffect(() => {
+    if (loggedIn) {
+      navigate('/', { replace: true });
+    }
+  }, [loggedIn]);
```

Comment:
- [ ] [Optional] This logic I strongly recommend you to be handled in the Login component. It is not a good thing to overload the App component with logic that its children can easily handle

Reply:
Hi @williamrolando88 I put this here to redirect to the login page after the logout process when we made it.